src/utils/utils.js

```diff
@@ -48,6 +48,12 @@ export const postComment = (endpoint, body, setPostedComment, setIsLoading) => {
     }).catch(err => console.log(err))
 }
 
+export const deleteItem = (endpoint, setIsLoading) => {
+    apiCall.remove(endpoint).then(() => {
+        setIsLoading(false)
+    }).catch(err => console.log(err))
```

Comment:
Consider how we could handle this if the apiCall were to fail? What would the user need to see?